src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2445,6 +2445,44 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
                                                 certificate_envelope=kv_cert_def)
 
 
+def sync_kv_cert(cmd, resource_group_name, name, certificate_name, key_vault):
+    Certificate = cmd.get_models('Certificate')
+    client = web_client_factory(cmd.cli_ctx)
+    webapp = client.web_apps.get(resource_group_name, name)
+    if not webapp:
+        err_msg = "'{}' app doesn't exist in resource group {}".format(name, resource_group_name)
+        raise ResourceNotFoundError(err_msg)
+    server_farm_id = webapp.server_farm_id
+    location = webapp.location
+
+    cert = client.certificates.get(resource_group_name, certificate_name, filter=f"ServerFarmId eq {'server_farm_id'}")
+
+    if cert is None:
+        no_cert_msg = "Cert with name {} is not associated with this site".format(certificate_name)
+        logger.warning(no_cert_msg)
+        return
```

Comment:
We currently only allow syncing to the latest version of the cert on AKV using the AKV apis to get the latest version Ex. https://docs.microsoft.com/en-us/rest/api/keyvault/getcertificates/getcertificates - so don't think we need a cert name at all here.